public/app/plugins/datasource/loki/components/stats.ts

```diff
@@ -52,3 +57,47 @@ export function shouldUpdateStats(
 
   return true;
 }
+
+export function getTimeRange(
```

Comment:
I am curious why isn't `getTimeRange` and `getStats` in here and they are not data source methods?